inspectors/view/qml/notation/notes/HeadSettings.qml

```diff
@@ -41,6 +41,27 @@ FocusableItem {
             noteHeadTypesModel: root.model ? root.model.noteheadTypesModel : null
         }
 
+        CheckBox {
+            isIndeterminate: {
+                if (!model)
+                    return false;
+                if (model.isUndefined)
+                    return false;
+
+                return model.parenthesesNotehead.value === NoteHead.PARENTHESIS_RIGHT || model.parenthesesNotehead.value === NoteHead.PARENTHESIS_LEFT
+            }
```

Comment:
How about `isIndeterminate: model && !model.isUndefined && (model.parenthesesNotehead.value === NoteHead.PARENTHESIS_RIGHT || model.parenthesesNotehead.value === NoteHead.PARENTHESIS_LEFT)`?

Reply:
That's what I did at the beginning, but at the end I found no reason to keep it as a one-liner.

Reply:
I think that's what's done in other inspector QMLs?

Reply:
I believe that clarity is more important